bibtasklets/bst_prodsync.py

```diff
@@ -93,6 +93,9 @@ def bst_prodsync(method='afs', with_citations='yes', with_claims='yes', skip_col
         write_message("Syncing records modified since %s" % last_run)
         with run_ro_on_slave_db():
             modified_records = intbitset(run_sql("SELECT id FROM bibrec WHERE modification_date>=%s", (last_run, )))
+            compacttime = last_run.replace('-', '').replace(' ', '').replace(':', '')
+            notimechangerecs = search_unit("%s->20250101000000" % compacttime, f='005', m='a')
```

Comment:
this will fail silently in 2025, but if this bibtasklet is still alive by then we will have other problems :)